src/main/java/com/urbanairship/datacube/DataCubeIo.java

```diff
@@ -94,18 +109,21 @@
      *                      the batch will not be flushed until the *next* write arrives after the timeout.
      */
     public DataCubeIo(DataCube<T> cube, DbHarness<T> db, int batchSize, long maxBatchAgeMs,
-                      SyncLevel syncLevel, String metricsScope) {
+                      SyncLevel syncLevel, String metricsScope, boolean perRollupMetrics) {
```

Comment:
What is the argument for making this configurable vs always enabled? Are we worried it would be too many metrics for prometheus or whatever you are using?

Reply:
Yeah, exactly that. I'm already getting pressure from ops to reduce metric load from other services, and I figure some cubes won't really benefit from this, or at least may only need it enabled temporarily.